ddtrace/ext/sql.py

```diff
@@ -23,13 +23,14 @@ def normalize_vendor(vendor):
     else:
         return vendor
 
+
 def parse_pg_dsn(dsn):
+    from psycopg2.extensions import parse_dsn
     """
     Return a dictionary of the components of a postgres DSN.
 
     >>> parse_pg_dsn('user=dog port=1543 dbname=dogdata')
     {"user":"dog", "port":"1543", "dbname":"dogdata"}
     """
-    # FIXME: replace by psycopg2.extensions.parse_dsn when available
```

Comment:
I think the `FIXME` didn't take in consideration the drop of all psycopg2 versions < 2.7